zipline/assets/asset_writer.py

```diff
@@ -358,7 +358,7 @@ class AssetDBWriter(object):
     """
     DEFAULT_CHUNK_SIZE = SQLITE_MAX_VARIABLE_NUMBER
 
-    @preprocess(engine=coerce_string_to_eng)
+    @preprocess(engine=coerce_string_to_eng(require_exists=False))
```

Comment:
Yup, this looks like the one place we don't want to require it exists. Do we ever append, as in, would we want to require that it does NOT exist?

Reply:
I'm not sure about this offhand.  @yankees714 do you know if we ever do inplace updates internally?  My inclination is to say that even if we don't expect to update an asset db inplace, it's possible that another user might, so I don't feel compelled to disallow that by construction.

Reply:
I'm not aware of any place where we append, but I agree that it probably makes sense to leave this flexible in case others want to.